exporter/splunkhecexporter/README.md

```diff
@@ -0,0 +1,53 @@
+# Splunk HTTP Event Collector (HEC) Exporter
+
+How to send metrics to a Splunk HEC endpoint. 
+
+The following configuration options are required:
+
+- `token` (no default): HEC requires a token to authenticate incoming traffic.
+- `endpoint` (no default): Splunk HEC URL.
+
+The following configuration options can also be configured:
+
+- `source` (no default): Optional Splunk source: https://docs.splunk.com/Splexicon:Source
+- `sourcetype` (no default): Optional Splunk source type: https://docs.splunk.com/Splexicon:Sourcetype
+- `index` (no default): Splunk index, optional name of the Splunk index targeted
+- `max_connections` (default: 100): Maximum HTTP connections to use simultaneously when sending data.
+- `disable_compression` (default: false): Whether to disable gzip compression over HTTP.
+- `timeout` (default: 10s): HTTP timeout when sending data.
+- `skiptlsverify` (default: false): Whether to skip checking the certificate of the HEC endpoint when sending data over HTTPS.
```

Comment:
Let's use `insecure_skip_verify` for consistency with other existing configs.

Reply:
Done.